### PR TITLE
ElectronTestApp e2e tests

### PR DESCRIPTION
**Pull request by derisen**
This PR adds e2e tests (AAD, ADFS) to ElectronTestApp sample using playwright. 

* Since the existing test utils are geared toward puppeteer types, I added another set of testUtils, leading to some seemingly duplicated code.
* I was able to run the tests in CI environment for macos and windows OSs, but currently having difficulty with linux systems via xvfb.

### CODE REVIEW

File: samples/msal-node-samples/ElectronTestApp/test/testUtils.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
+import { ElectronApplication, Page } from "playwright";
```
Comment (tnorling) [inline samples/msal-node-samples/ElectronTestApp/test/testUtils.ts]: Would be great if the contents of this file could live in the e2eTestUtils folder so we can reuse for our other tests

Comment (derisen) [reply samples/msal-node-samples/ElectronTestApp/test/testUtils.ts]: of course -moved it into e2eTestUtils

**Review by hectormmg — APPROVED** [review]
These look great @derisen ! Thanks!

File: .github/workflows/msal-node-e2e.yml [hunk 2]
```diff
@@ -105,3 +105,95 @@ jobs:
...
+
+    name: ${{ matrix.sample }} - ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: 14
```
Comment (tnorling) [inline .github/workflows/msal-node-e2e.yml]: Reason why we only need to test against one version of node? If we need to choose one I'd suggest 16 or 18 as we're going to start enforcing 16+ for local development of our repo soon

File: .github/workflows/msal-node-e2e.yml [hunk 3]
```diff
@@ -105,3 +105,95 @@ jobs:
...
+      run: |
+        npm run install:local
+        npm install
+      env:
+        PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: 1
+
+    - name: Install Playwright
+      working-directory: samples/msal-node-samples/${{ matrix.sample }}
+      run: |
+        npx playwright install --with-deps chromium
```
Comment (tnorling) [inline .github/workflows/msal-node-e2e.yml]: Can we include this as a postinstall step in the package.json instead?

**Review by tnorling — APPROVED** [review]
Approved with comments

File: .github/workflows/msal-node-e2e.yml (hunk 2 above)
Comment (derisen) [reply .github/workflows/msal-node-e2e.yml]: Electron itself is packaged with a particular version of Node, so I didn't see much value in testing over a range of node versions. Let me switch to v16 in that case.

File: .github/workflows/msal-node-e2e.yml (hunk 3 above)
Comment (derisen) [reply .github/workflows/msal-node-e2e.yml]: will try 👍

**Pull request merged**